### ISSUE TITLE
Talisman: Digital Edition (247000)

### ISSUE THREAD

**Issue by jvalrog** [ValveSoftware/Proton#903]
[My specs](https://gist.github.com/jvalrog/93c304fabb18142bbec0c6e839db06f9)

### What works:

* In-game menus
* Fullscreen/windowed
* Alt-tabbing
* Asmodee login
* Steam overlay
* Language selection
* In-game store menu
* Normal games
* Quitting

### What doesn't work:

* Showing the Steam overlay makes the game lose the focus, having to click in it to regain it.

**Comment by SteveHeist** [comment]
Yeah, I'm gonna have to say it's not ready. I'm on AMD hardware and get nothing but a heavily borked image for a half-second and a crash-out. Tried 3.7-6 & 3.7-7 Beta - and it does the same thing through Lutris.

**Comment by magos-linux** [comment]
CPU: Intel(R) Core(TM) i7-4765T RAM: 16G SSD: KINGSTON SUV300S37A120G
Video: NVIDIA GeForce GTX 960 4G (nvidia-current 396.54)
OS: MagOS Linux 2016.64
DE: Plasma+compiz-fusion 1680x1050
Proton version: 3.7-8 3.16-4beta

## Issues
  - Game crashes after black screen within some seconds on proton 3.7-8
  - Game crashes immediately on 3.16-4beta

**Comment by tingvarsson** [comment]
Hi, the game got an updated engine (dx11 instead of what I presume is dx9) during October (https://steamcommunity.com/app/247000/discussions/0/3145094199290301116/)

And is not working well anymore:
- Crashes with dxvk at start (tried to collect an apitrace but it failed for unclear reason, generated only a 2 byte trace file).
- Graphical glitches when using wined3d instead
- No dx9 fallback when disabled dx11 all together

Note: I've only tested it on 3.16-4.

The old engine can still be used (uncertain for how long, and it wont get updates) by opting into one of the beta releases. That is how I confirmed that it was not using dx11 before. That version worked out of the box for me, similar as the original topic here.

**Comment by lucifertdark** [comment]
I got the game to run by adding the launch option PROTON_USE_WINED3D=1 %command% it's not a pretty sight though & I wouldn't want to try playing it like that.

**Comment by kisak-valve** [comment]
Can someone with the game please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by tingvarsson** [comment]
[steam-247000.log](https://github.com/ValveSoftware/Proton/files/2565832/steam-247000.log)

**Comment by magos-linux** [comment]
[steam-247000.log](https://github.com/ValveSoftware/Proton/files/2566043/steam-247000.log)

**Comment by tingvarsson** [comment]
Fyi. Seems game got updated. Works in dx11 with dxvk now.

**Comment by kaincenteno** [comment]
works great for me now. Can someone else confirm?
